cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/messages/CassandraStoredMessageFilter.java

```diff
@@ -79,13 +79,10 @@ public CassandraStoredMessageFilter(PageId pageId, String sessionAlias, String d
 	public Select addConditions(Select select) {
 		select = select
 				.whereColumn(FIELD_BOOK).isEqualTo(bindMarker())
+				.whereColumn(FIELD_PAGE).isEqualTo(bindMarker())
 				.whereColumn(FIELD_SESSION_ALIAS).isEqualTo(bindMarker())
 				.whereColumn(FIELD_DIRECTION).isEqualTo(bindMarker());
 
-		if (page != null) {
-			select = select.whereColumn(FIELD_PAGE).isEqualTo(bindMarker());
-		}
```

Comment:
Maybe we should check in the constructor that the page is not null to show that it MUST NOT be null